### DISCUSSION TITLE
FINN tutorial @ FPL'24

### DISCUSSION THREAD

**Discussion by auphelia** [Xilinx/finn#1144] [Announcements]
We are excited to announce that there will be a FINN tutorial also at [this year's FPL in Turin](http://asaclab.polito.it/fpl2024/)! 🥳 

For this year, we decided to change the format of the community section. We will give members of our community the chance to present their work with FINN during a poster session. Please see for more details about the poster session the post of my colleague @lstasytis below.

The tutorial will be a full-day in-person event held on Tuesday, September 3rd at FPL. The tutorial will consist of an introduction to the FINN framework, a poster session with contributions from the community and a hands-on lab. 
For the hands-on lab, we offer two options: 
-	Participants choosing the “Beginner” option are introduced to the FINN ecosystem by training a quantized example network with Brevitas and deploying it with FINN.
-	Participants that choose the “Advanced” option will have the opportunity to do a hands-on exploration of the new features in the FINN compiler and their implied additional builder arguments for customizing the resulting architecture that can accelerate neural network inference.

Looking forward to seeing you in Italy!

**Comment by lstasytis** [comment]
**Call for Posters to Present at FPL24’s FINN Tutorial Session**

At this year’s [FPL’24 conference in Turin, Italy](http://asaclab.polito.it/fpl2024/), as part of the FINN tutorial, the team will additionally be hosting a FINN Poster Session and so we would like to invite you all to submit posters highlighting your FINN-related work which you would like to present!
You have two options for how your poster may be presented:

- In person during the tutorial by bringing in a printed version
- Online on the FINN project page

The poster should highlight either how you have used FINN for your own use-cases or the contributions you have made to the FINN project itself, for example - features which have already been integrated into FINN or are still in progress.

**Poster requirements:**
Size: A1 (594 mm x 841 mm)
Orientation: Vertical
Example templates which can be used as a starting point are attached to this post in .drawio and .ppt file formats: [FPL_poster_templates_2024.zip](https://github.com/user-attachments/files/16414903/FPL_poster_templates_2024.zip)

**Important Dates:**
Conference dates: September 2nd – September 6th
FINN Tutorial date: September 3rd
Call For Posters Begins: July 29th, 2024
Call for Posters Ends: **August 19th, 2024, 23:59:59 (AoE)**
Announcement of Acceptance: August 26th, 2024

**How to submit:**
Please send your posters in .pdf format to @auphelia using the email address: [jakoba.petri-koenig@amd.com](mailto:jakoba.petri-koenig@amd.com) with a subject header “FINN-Poster-FPL24” and let us know if you will be attending the tutorial in-person.
